tests/unit/rdf/test_resource_serialisation.py

```diff
@@ -314,6 +315,175 @@ class A(NewResource):
         URIRef("http://some-entity-uri"),
     ) in graph
 
+def test_markdown_datatype():
+    """ """
+
+    class A(NewResource):
+        p: Annotated[
+            str,
+            Triple(
+                URIRef("http://original-uri"),
+                PropertyStatus.recommended,
+                map_str_to_markdown,
+            ),
+        ]
+    a = A("http://some-a-uri")
+    a.p = "Hello, World"
+
+    graph = a.to_graph(Graph())
+
+    assert len(graph) == 2
+    assert (
+        URIRef("http://some-a-uri"),
+        URIRef("http://original-uri"),
+        Literal("Hello, World", datatype="https://www.w3.org/ns/iana/media-types/text/markdown#Resource"),
+    ) in graph
+
+    assert (
+        URIRef("http://some-a-uri"),
+        RDF.type,
+        RDFS.Resource,
+    ) in graph
+
+def test_italic_detected_as_html(capsys):
```

Comment:
I think this (and the following test names) are misleading since they suggest that markdown syntax should be detected as HTML when they're testing for the opposite (as they should be).